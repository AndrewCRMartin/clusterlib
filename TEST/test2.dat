 1  5  6  3  5  3  6  8  9  5 
 1  5  6  3  5  3  6  8  9  5 
 5  6  7  5  3  4  5  6  8  3 
 5  6  7  5  4  5  3  5  7  1 
 3  4  5  6  7  8  6  5  6  4 
 5  6  5  6  7  4  5  6  4  3 
 4  5  6  7  4  5  6  4  3  8 
 7  8  7  6  7  8  6  4  5  5 
 6  4  3  4  5  7  8  9  2  3 
 4  3  5  4  5  6  7  6  5  3 
 5  7  4  6  7  4  6  3  2  3 
 4  6  7  8  5  6  8  5  4  3 
 4  2  3  2  4  5  6  7  8  3 