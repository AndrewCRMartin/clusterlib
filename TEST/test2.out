Vector   :   0   1   2   3   4   5   6   7   8   9  10  11  12
Clust13  :   0   1   2   3   4   5   6   7   8   9  10  11  12
Clust12  :   0   0   2   3   4   5   6   7   8   9  10  11  12 (0.000)
Clust11  :   0   0   2   2   4   5   6   7   8   9  10  11  12 (2.828)
Clust10  :   0   0   2   2   4   5   6   7   8   9  10   6  12 (3.317)
Clust9   :   0   0   2   2   4   5   6   7   8   9   5   6  12 (4.000)
Clust8   :   0   0   2   2   4   5   6   7   8   4   5   6  12 (4.123)
Clust7   :   0   0   2   2   4   5   5   7   8   4   5   5  12 (4.472)
Clust6   :   0   0   2   2   4   5   5   7   8   4   5   5   4 (4.690)
Clust5   :   0   0   2   2   4   4   4   7   8   4   4   4   4 (5.000)
Clust4   :   0   0   2   2   2   2   2   7   8   2   2   2   2 (5.385)
Clust3   :   0   0   2   2   2   2   2   7   2   2   2   2   2 (5.385)
Clust2   :   0   0   2   2   2   2   2   2   2   2   2   2   2 (5.568)
Clust1   :   0   0   0   0   0   0   0   0   0   0   0   0   0 (5.745)
